bin/console

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env php
 <?php
 
+if (getenv('ENV_DIR') !== false) {
```

Comment:
nit: if we do `ENV_DIR= bin/console`, the test will pass. I'd also test for `!== ''`